### ISSUE TITLE
Scrolling in both directions required to view contents in reflow mode

### ISSUE THREAD

**Issue by jose-cds** [cds-snc/platform-forms-client#3098]
**HC Label: Serious**

Pages:  Advancing Accessibility Standards Research Program Expression of Interest
URL: https://forms-staging.cdssandbox.xyz/en/id/134
User agent:  Chrome (v119.0.6045.200)
Language: English and French

Issue Description: 
When content is reflowed, users have to scroll in both directions horizontally and vertically to view some contents on the pages.

Steps to reproduce:

1. Open the above URL in the Chrome browser
2. Set the desktop browser to 1280 width x 1024 height and zoom in to 400% (Chrome -> Web Developer settings -> Resize Window to 1280x1024)
3. Visual check of the contents on the pages

Expected result: When content is reflowed, it should be presented without loss of information or functionallity and not requiring scrolling in two dimensions to view its contents.

**Comment by srtalbot** [comment]
Being addressed as part of conditional logic
